lib/api/controllers/funnelController.js

```diff
@@ -398,6 +414,29 @@ class FunnelController {
       });
   }
 
+  /**
+   * @param {KuzzleRequest} request
+   * @param {String} prefix
+   * @return {Promise<KuzzleRequest>}
+   */
+  performDocumentAlias(request, prefix) {
+    const { controller, action } = request.input;
+    const mustTrigger = controller === 'document'
+    && this.documentEventAliases.mirrorList[action]
+    && (prefix !== 'before' || !this.documentEventAliases.notBefore.includes(action));
+    
+    if (!mustTrigger) {
+      return Bluebird.resolve(request);
+    }
+
+    const alias = this.documentEventAliases.mirrorList[action];
+    const event = `${this.documentEventAliases.namespace}:${prefix}${capitalizeFirstLetter(alias)}`;
+    const extractor = new DocumentExtractor(request);
+    return this.kuzzle
+      .pipe(event, extractor.extract(), request)
+      .then((documents) => extractor.insert(documents));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3524)